python/contactflowhelper/README.md

```diff
@@ -0,0 +1,251 @@
+# Contact Flow Helper
```

Comment:
suggestions - can you provide examples of how customers can use these? some are straightforward, but others i'm interested in what use cases customers have for them (like splitting the text). providing examples can help customers think of other use cases they may not have considered before.

Reply:
Done